packages/daimo-api/src/server/trpc.ts

```diff
@@ -34,6 +34,46 @@ function getHeader(h: string | string[] | undefined) {
   else return h || "";
 }
 
+export type DaimoVersion = {
```

Comment:
can use semver (`npm i semver @types/semver`, i think) then we don't need our own parser